### ISSUE TITLE
Is it possible to make fixuid "silent"

### ISSUE THREAD

**Issue by ruzickap** [boxboat/fixuid#11]
Hello guys.

It looks like `fixuid` is writing it's output into `stderr`:

```
$ fixuid > /dev/null
fixuid: fixuid should only ever be used on development systems. DO NOT USE IN PRODUCTION
fixuid: updating user 'docker' to UID '1000
fixuid: updating group 'docker' to GID '1000
fixuid: recursively searching path /
fixuid: chown /home/docker
fixuid: chown /home/docker/.bash_logout
fixuid: chown /home/docker/.bashrc
fixuid: chown /home/docker/.profile
```

Can you add some parameter to make `fixuid` run silently (like: `--quiet` or `-q`) ?

Thank you

**Comment by ruzickap** [comment]
Thanks for adding `-q` option.

Any chance to release it (maybe `0.4`) soon ?

**Comment by caleblloyd** [comment]
Thanks for the report.  I think this will be helpful for containers used as command-line tools that incorporate `fixuid`.

Just shipped in [0.4](https://github.com/boxboat/fixuid/releases/tag/v0.4)

**Comment by ruzickap** [comment]
Thanks a lot for a quick response + release...
